### PR TITLE
Add app

### PR DESCRIPTION
**Pull request by grantwforsythe**
Add this new feature that calls a request to API to print random names.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import requests
+
+
+def get_name(n: int) -> str:
```
Comment (gwforsythe) [inline app.py]: Can we rename the function to get names?

File: app.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    """ Get a random name """
+    for _ in range(n):
+        response = requests.get('https://api.namefake.com/')
+        yield response.json()['name']
+
+
+if __name__ == '__main__':
+    for name in get_name(5):
```
Comment (gwforsythe) [inline app.py]: Can we declare the constant as a variable?

File: app.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import requests
+
+
+def get_name(n: int) -> str:
+    """ Get a random name """
+    for _ in range(n):
+        response = requests.get('https://api.namefake.com/')
+        yield response.json()['name']
```
Comment (gwforsythe) [inline app.py]: Can we add error handling to ensure that the response was made successfully?
```suggestion
        response = requests.get('https://api.namefake.com/')
        if response.ok:
            yield response.json()['name']
        else:
            response.raise_for_status()
```

**Review by gwforsythe — CHANGES_REQUESTED** [review]
Thanks for adding this feature! I just left a few comments for you to view

File: app.py (hunk 1 above)
Comment (grantwforsythe) [reply app.py]: Yes, I think renaming the function would certainly make it more descriptive

File: app.py (hunk 2 above)
Comment (grantwforsythe) [reply app.py]: I agree that the function should be declared as a variable

**Review by gwforsythe — APPROVED** [review]
Approved!

**Pull request merged**